tests/posix_close/send_alarms.c

```diff
@@ -0,0 +1,66 @@
+#include <pthread.h>
+#include <stdlib.h>
+#include <signal.h>
+#include <string.h>
+#include <stdio.h>
+
+#include "warnp.h"
+
+#include "send_alarms.h"
+
+struct send_alarms {
+	pthread_t to_thr;
+	volatile int * stop;
+};
+
+static void *
+workthread_send_alarm(void * cookie)
+{
+	struct send_alarms * SA = cookie;
+
+	while (*SA->stop == 0) {
+		if (pthread_kill(SA->to_thr, SIGALRM)) {
```

Comment:
I'm not at all clear about whether this should be `pthread_kill()` or `raise()`.